app/demoservice/libs/demos.py

```diff
@@ -14,7 +14,7 @@
 from demoservice.logging import get_demo_logger
 
 
-DEMO_PR_URL_TEMPLATE = '{repo_name}-pr-{github_pr}.run.demo.haus'
+DEMO_PR_URL_TEMPLATE = '{org_name}-{repo_name}-pr-{github_pr}.run.demo.haus'
```

Comment:
I had a quick informal poll with some of the team and there was a preference to keep the repo name at the beginning. Then as the repo is usually the production domain, the URL can easily be trimmed.

So this is prefrerred

```python3
DEMO_PR_URL_TEMPLATE = '{repo_name}-{org_name}-pr-{github_pr}.run.demo.haus'
```

Then you can easily trim the middle of the URL to get the production URL:
www.ubuntu.com ~~-canonical-websites-pr-1.run.demo.haus~~ /example/url